Formula/wego.rb

```diff
@@ -0,0 +1,38 @@
+require "language/go"
+
+class Wego < Formula
+  desc "Weather app for the terminal"
+  homepage "https://github.com/schachmat/wego"
+  url "https://github.com/schachmat/wego/archive/2.0.tar.gz"
+  sha256 "d63d79520b385c4ed921c7decc37a0b85c40af66600f8a5733514e04d3048075"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/mattn/go-colorable" do
+    url "https://github.com/mattn/go-colorable.git",
+    :revision => "ed8eb9e318d7a84ce5915b495b7d35e0cfe7b5a8"
```

Comment:
Is that the new norm? Note that before I submitted, I checked the existing golang formulae, ordered by last update. The three most recent ones — `acmetool`, `termshare` and `gitlab-ci-multi-runner` — all have different indentations... So I just went with what `gdm` emitted (which is consistent with `termshare`, last updated yesterday).